pkg/labels/arraylist.go

```diff
@@ -40,3 +40,16 @@ func (ls LabelArrayList) GetModel() [][]string {
 	}
 	return res
 }
+
+// Same returns true if the label arrays lists have the same label arrays in the same order.
+func (ls LabelArrayList) Same(b LabelArrayList) bool {
```

Comment:
Could this be named `Equal`? Can you add unit tests for this? what should be the result of this function for a non-nil received with a nil argument?

Reply:
I named it `Same` to be consistent with the function is also called `Same` on the `LabelArray`. It is called `Equal` on `Label`.

Thanks for pointing out the missing unit tests. I actually wrote unit-tests for this, but apparently forgot to add them to the commit.

Reply:
(Should they both be called `Equals` / `Equals()`? `if foo.Same(bar) { ... }` doesn't read terribly easily..)